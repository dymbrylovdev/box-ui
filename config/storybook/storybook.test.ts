import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  // your own configuration
  configPath: './config/storybook',
  framework: 'react',
});
