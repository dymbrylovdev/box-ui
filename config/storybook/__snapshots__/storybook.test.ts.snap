// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Shared/AppLink Primary Dark 1`] = `
<div
  className="app light"
>
  <div
    className="app dark"
  >
    <a
      className="appLink  primary"
      href="/"
      onClick={[Function]}
    >
      PRIMARY
    </a>
  </div>
</div>
`;

exports[`Storyshots Shared/AppLink Primary Light 1`] = `
<div
  className="app light"
>
  <div
    className="app light"
  >
    <a
      className="appLink  primary"
      href="/"
      onClick={[Function]}
    >
      PRIMARY
    </a>
  </div>
</div>
`;

exports[`Storyshots Shared/AppLink Revert Primary Light 1`] = `
<div
  className="app light"
>
  <div
    className="app light"
  >
    <a
      className="appLink  revert_primary"
      href="/"
      onClick={[Function]}
    >
      PRIMARY
    </a>
  </div>
</div>
`;

exports[`Storyshots Shared/AppLink Revert Secondary Dark 1`] = `
<div
  className="app light"
>
  <div
    className="app dark"
  >
    <a
      className="appLink  revert_secondary"
      href="/"
      onClick={[Function]}
    >
      PRIMARY
    </a>
  </div>
</div>
`;

exports[`Storyshots Shared/AppLink Secondary Dark 1`] = `
<div
  className="app light"
>
  <div
    className="app dark"
  >
    <a
      className="appLink  secondary"
      href="/"
      onClick={[Function]}
    >
      SECONDARY
    </a>
  </div>
</div>
`;

exports[`Storyshots Shared/AppLink Secondary Light 1`] = `
<div
  className="app light"
>
  <div
    className="app light"
  >
    <a
      className="appLink  secondary"
      href="/"
      onClick={[Function]}
    >
      SECONDARY
    </a>
  </div>
</div>
`;

exports[`Storyshots Widgetes/ErrorPage Error Page Dark 1`] = `
<div
  className="app light"
>
  <div
    className="app dark"
  >
    <div
      className="ErrorPage "
    >
      <p>
        Произошла непредвиденная ошибка
      </p>
      <button
        className="Button size_m undefined "
        onClick={[Function]}
        type="button"
      >
        Обновить страницу
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots Widgetes/ErrorPage Error Page Light 1`] = `
<div
  className="app light"
>
  <div
    className="ErrorPage "
  >
    <p>
      Произошла непредвиденная ошибка
    </p>
    <button
      className="Button size_m undefined "
      onClick={[Function]}
      type="button"
    >
      Обновить страницу
    </button>
  </div>
</div>
`;

exports[`Storyshots Widgetes/Sidebar Dark 1`] = `
<div
  className="app light"
>
  <div
    className="app dark"
  >
    <div
      className="sidebar "
      data-testid="sidebar"
    >
      <div
        className="navigation_container"
      >
        <a
          className="appLink navigation revert_primary"
          href="/"
          onClick={[Function]}
        >
          <div />
          <span
            className="link"
          >
            MAIN
          </span>
        </a>
        <a
          className="appLink navigation revert_primary"
          href="/cart"
          onClick={[Function]}
        >
          <div />
          <span
            className="link"
          >
            CARD
          </span>
        </a>
      </div>
      <div
        className="theme"
      >
        <button
          className="Button size_xl clear  "
          onClick={[Function]}
          type="button"
        >
          <div />
        </button>
        <button
          className="Button size_l backgroundInverted  "
          onClick={[Function]}
          type="button"
        >
          LANG
        </button>
      </div>
      <button
        className="Button size_xl backgroundInverted btmTheme"
        data-testid="toggle-sidebar"
        onClick={[Function]}
        type="button"
      >
        &lt;
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots Widgetes/Sidebar Light 1`] = `
<div
  className="app light"
>
  <div
    className="sidebar "
    data-testid="sidebar"
  >
    <div
      className="navigation_container"
    >
      <a
        className="appLink navigation revert_primary"
        href="/"
        onClick={[Function]}
      >
        <div />
        <span
          className="link"
        >
          MAIN
        </span>
      </a>
      <a
        className="appLink navigation revert_primary"
        href="/cart"
        onClick={[Function]}
      >
        <div />
        <span
          className="link"
        >
          CARD
        </span>
      </a>
    </div>
    <div
      className="theme"
    >
      <button
        className="Button size_xl clear  "
        onClick={[Function]}
        type="button"
      >
        <div />
      </button>
      <button
        className="Button size_l backgroundInverted  "
        onClick={[Function]}
        type="button"
      >
        LANG
      </button>
    </div>
    <button
      className="Button size_xl backgroundInverted btmTheme"
      data-testid="toggle-sidebar"
      onClick={[Function]}
      type="button"
    >
      &lt;
    </button>
  </div>
</div>
`;

exports[`Storyshots shared/Button Background 1`] = `
<div
  className="app light"
>
  <div
    className="app dark"
  >
    <button
      className="Button size_m background "
      type="button"
    >
      Click
    </button>
  </div>
</div>
`;

exports[`Storyshots shared/Button Background Inverted 1`] = `
<div
  className="app light"
>
  <button
    className="Button size_m backgroundInverted "
    type="button"
  >
    Click
  </button>
</div>
`;

exports[`Storyshots shared/Button Clear 1`] = `
<div
  className="app light"
>
  <button
    className="Button size_m clear "
    type="button"
  >
    Click
  </button>
</div>
`;

exports[`Storyshots shared/Button Clear Inverted 1`] = `
<div
  className="app light"
>
  <button
    className="Button size_m clearInverted "
    type="button"
  >
    Click
  </button>
</div>
`;

exports[`Storyshots shared/Button Disabled 1`] = `
<div
  className="app light"
>
  <button
    className="Button size_m backgroundInverted disabled "
    disabled={true}
    type="button"
  >
    Click
  </button>
</div>
`;

exports[`Storyshots shared/Button Outline 1`] = `
<div
  className="app light"
>
  <button
    className="Button size_m outline "
    type="button"
  >
    Click
  </button>
</div>
`;

exports[`Storyshots shared/Button Primary L 1`] = `
<div
  className="app light"
>
  <button
    className="Button size_l undefined "
    type="button"
  >
    Click
  </button>
</div>
`;

exports[`Storyshots shared/Button Primary XL 1`] = `
<div
  className="app light"
>
  <button
    className="Button size_xl undefined "
    type="button"
  >
    Click
  </button>
</div>
`;

exports[`Storyshots shared/Button Square 1`] = `
<div
  className="app light"
>
  <button
    className="Button square size_m backgroundInverted "
    type="button"
  >
    &gt;
  </button>
</div>
`;

exports[`Storyshots shared/Button Square L 1`] = `
<div
  className="app light"
>
  <button
    className="Button square size_l backgroundInverted "
    type="button"
  >
    &gt;
  </button>
</div>
`;

exports[`Storyshots shared/Button Square XL 1`] = `
<div
  className="app light"
>
  <button
    className="Button square size_xl backgroundInverted "
    type="button"
  >
    &gt;
  </button>
</div>
`;

exports[`Storyshots shared/Loader Primary 1`] = `
<div
  className="app light"
>
  <div
    className="lds_roller  primary"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;

exports[`Storyshots shared/Loader Secondary 1`] = `
<div
  className="app light"
>
  <div
    className="lds_roller  secondary"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
